DRAFT.md

```diff
@@ -489,6 +491,15 @@ With some suitably compelling motivation, the `explicit` keyword could be
 removed from some constructors in a later revision of the C++ standard without
 rendering code ill-formed.
 
+### The `valueless_after_move` member function
+
+Both `indirect` and `polymorphic` have a valueless state that is used to query
+the object state. By intention, this member function should not be often called:
```

Comment:
Intentionally, this member function should not often be called:

Reply:
Dropped "Intentionally" altogether - it implies other elements of our design may be accidents :-)